io_util.py

```diff
@@ -305,3 +305,56 @@ def untar_input_file(tar_file_path: str) -> str:
         tar.extractall(path=path, filter="data")  # type: ignore
         filename = tar.getmembers()[0].name
     return path + f"/{filename}"
+
+
+# makes sure the model is available locally, if not download it from S3, if that fails download from Huggingface
+def check_model_availability() -> bool:
+    logger = logging.getLogger(__name__)
+    if os.path.exists(cfg.FILE_SYSTEM.BASE_MOUNT_MODEL + "/model.bin"):
+        logger.info("Model found locally")
+        return True
+    else:
+        logger.info("Model not found locally, attempting to download from S3")
+        if not validate_s3_uri(cfg.WHISPER_ASR_SETTINGS.MODEL):
+            logger.error("No S3 URI detected")
```

Comment:
Preferably not an error but an info log: it is perfectly sane behaviour to not find an S3 URI, right?